doc/appendices/command-line/traffic_layout.en.rst

```diff
@@ -60,6 +60,12 @@ Options:
 
  Force create sandbox and overwrite existing directory 
 
+ 4. relocate flag for creating: ::
+
+    traffic_runroot --relocate --init /path/to/sandbox
```

Comment:
You need to review all the text. The code that was traffic_runroot is not traffic_layout. 

I think we need to review all text. I would suggest you install Grammarly (https://www.grammarly.com/) and run all the text through that and then have us review it again.

Reply:
This Grammarly is a really nice thing to use